platforms/corda-ent/configuration/roles/setup/notary/tasks/main.yaml

```diff
@@ -56,8 +68,8 @@
     db_username: "{{ org.services.notary.name }}-db-user"
     db_password: "{{ org.services.notary.name }}-db-password"
     db_port: "{{ org.services.notary.dbtcp.port }}"
-    networkmap_url: "{{ network | json_query('orderers[?type==`networkmap`].uri') | first }}"
-    idman_url: "{{ network | json_query('orderers[?type==`idman`].uri') | first }}"
+    idman_url: "http://{{ org.services.idman.name }}.{{ org.name | lower }}-ent:{{ org.services.idman.port }}"
```

Comment:
It worked for the p2p address in notary initial registration job, not here. Mostly because of the identity associated with the ambassador certificate.

Reply:
The Urls works though on the exposed enpoints.